src/parameterizations/lateral/MOM_internal_tides.F90

```diff
@@ -578,7 +614,8 @@ subroutine propagate_int_tide(h, tv, cn, TKE_itidal_input, vel_btTide, Nb, dt, &
         itidal_loss_mode(i,j)       = itidal_loss_mode(i,j) + CS%TKE_itidal_loss(i,j,a,fr,m)
         allprocesses_loss_mode(i,j) = allprocesses_loss_mode(i,j) + &
                                  CS%TKE_leak_loss(i,j,a,fr,m) + CS%TKE_quad_loss(i,j,a,fr,m) + &
-                                 CS%TKE_itidal_loss(i,j,a,fr,m) + CS%TKE_Froude_loss(i,j,a,fr,m)
+                                 CS%TKE_itidal_loss(i,j,a,fr,m) + CS%TKE_Froude_loss(i,j,a,fr,m) + &
```

Comment:
Please be sure to add parentheses specifying the order of sums of 3 or more terms, here and elsewhere in this module.
The expressions `A+(B+C)` and `(A+B)+C` are not the same at the level of roundoff, and the differences can be very significant if they are of opposite signs and diversion magnitudes.  Consider the sum of 1, -1 and 1e-20, which for double precision floating point numbers could give 0 or 1e-20, depending on the order in which the sums are done. 
We have often found it very helpful for debugging enforce the order of these sums with parentheses.  (In fact, we are thinking of using Marshall's Flint capabilities to detect and replace every instance where the code does not specify the order of 2 or more sums.)